assets/app/features/upgrade/templates/openstack-services.jade

```diff
@@ -4,7 +4,7 @@ p(translate='') upgrade.steps.openstack-services.description
 button.btn.btn-success.center-block(
     ng-click='upgradeOpenStackServicesVm.openStackServices.stopServices()',
     ng-disabled='upgradeOpenStackServicesVm.checks.services.running || \
-        upgradeOpenStackServicesVm.checks.backup.completed || upgradeOpenStackServicesVm.checks.backup.running',
```

Comment:
This is not a design change right? This is a leftover from https://github.com/crowbar/crowbar-ui/pull/134/ ?